### DISCUSSION TITLE
semantic-kernel on promptflow

### DISCUSSION THREAD

**Discussion by mylee04** [microsoft/semantic-kernel#4936] [Q&A] [answered]
Hello, I'm trying to figure out how to utilize semantic kernel on promptflow. Do you have any tutorial for this? for example, 
```
kernel = sk.Kernel()
kernel.add_chat_service(
    "chat_completion",
     sk.AzureChatCompletion(
         deployment_name=AZURE_OPEN_AI_CHAT_COMPLETION_DEPLOYMENT_NAME,
         endpoint=AZURE_OPEN_AI_ENDPOINT,
         api_key=AZURE_OPEN_AI_API_KEY,     ),)
```

here, I don't need to use AzureChatCompletion part in promptflow, they have all these things if I make a llm node,

**Comment by harbhajan21** [comment, marked as answer]
You can just store the credentials of Azure Open AI in .env file and later you can retrieve it. Here, I'm attaching the YouTube video link for the reference.
Link: https://www.youtube.com/watch?v=90hhJHTWz50&t=13s

**Reply by mylee04** [reply]
Thank you, and have you ever tried it on promptflow?

**Reply by harbhajan21** [reply]
Yes, I tried and worked on  promptflow.

**Reply by mylee04** [reply]
Hmm, okay, I will try that too. But I didn't solve it yet.
